packages/app/src/update/reducers/update-os.reducer.test.ts

```diff
@@ -144,128 +146,167 @@ describe("startUpdateOs action", () => {
 })
 
 describe("checkForUpdate", () => {
-  describe.each([true, false])(
-    "when silent update flag equals to %p",
-    (silentUpdateFlag) => {
-      test("pending action sets proper states and clears error", () => {
-        expect(
-          updateOsReducer(
-            {
-              ...initialState,
-              error: exampleError,
-            },
-            {
-              type: pendingAction(UpdateOsEvent.CheckForUpdate),
-              meta: {
-                arg: {
-                  isSilentCheck: silentUpdateFlag,
-                },
+  const testCases: CheckForUpdateMode[] = [
+    "normal",
+    "silent-check",
+    "try-again",
```

Comment:
why no enum? 🤔 It will be easy to maintain / reactor variables with an enum 😉

Reply:
Changed in https://github.com/mudita/mudita-center/pull/1085/commits